docs/development/documentation.md

```diff
@@ -0,0 +1,26 @@
+# Documentation
+
+Sphinx est utilisé pour la génération de la documentation depuis des pages écrites en Markdown (via le [parser MyST](https://myst-parser.readthedocs.io/en/latest/)).
+
+## Génération du site web de documentation
+
+Pour la génération:
+
+```bash
+# install aditionnal dependencies
+python -m pip install -U -r requirements/documentation.txt
```

Comment:
We also need to install base requirements.txt.

Can you move the requirements.txt file to the requirements folder and add it in the documentation for documentation build ?

The base requirements.txt also need to be updated to use the same packages as the ones described in docker/debian/Dockerfile. (error with psycopg2, we need to use psycopg2-binary